R/set_model.R

```diff
@@ -0,0 +1,34 @@
+# set_model---------------------------------------------------------------------
+#' This function allows to change the model path of an experiment object
+#'
+#' @param model Path of new gaml model file
+#' @param exp Experiment in question
+#'
+#' @examples
+#' exp1 <- load_experiment("sir", system.file("examples", "sir.gaml",
+#'                         package = "rama"))
+#' model(exp1)
+#' set_model(exp1, system.file("examples", "CopyOfsir.gaml",
+#'                         package = "rama"))
+#' model(exp1)
+#'
+#' @export
+
+set_model <- function(model, exp){
```

Comment:
Can you write a `model<-.experiment` method for that?
Meaning that you'd need to declare the corresponding generic `model<-` too.

Reply:
done